### DISCUSSION TITLE
AI 驱动的 P 图工具

### DISCUSSION THREAD

**Discussion by adoyle-h** [just-talks/tech-talks#15] [3.分享创造]
https://github.com/Sanster/lama-cleaner

它提供 [Docker](https://lama-cleaner-docs.vercel.app/install/docker)，也有[在线版](https://huggingface.co/spaces/Sanster/Lama-Cleaner-lama)，试了一下效果相当好。它可以选择用 CPU 计算还是用 GPU 计算。

用的 SOTA (State-of-the-Art) 模型，我也不懂 AI 算法，看起来很厉害的样子。

**Comment by zhmou** [comment]
SOTA模型不是某一种模型，这个单词的缩写就是指某一细分任务（比如对于CV领域的分类、识别、语义分割）表现最优的模型。
一般一篇AI论文都要和之前的SOTA模型做对比，以体现proposed method的优点
